### PR TITLE
pythonPackages.pytmx: 3.24.0 -> 3.25.0

### PR DESCRIPTION
**Pull request by oxzi**
###### Motivation for this change
Version bump to the latest release.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 121485` at 10525ace run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages built successfully:</summary>
  <ul>
    <li>python38Packages.pytmx</li>
    <li>python39Packages.pytmx</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unclear-gpl.md">unclear-gpl</a>

`lgpl3` is a deprecated license, please check if project uses `lgpl3Plus` or `lgpl3Only` and change `meta.license` accordingly.

Near pkgs/development/python-modules/pytmx/default.nix:31:5:
```
   |
31 |     license = licenses.lgpl3;
   |     ^
```
</li>
  </ul>
</details>

**Review by dotlambda — CHANGES_REQUESTED** [review]
License should be `lgpl3Plus`.

Would be nice:
```nix
pythonImportsCheck = [
  "pytmx.pytmx"
  "pytmx.util_pygame"
  "pytmx.util_pyglet"
  "pytmx.util_pysdl2"
];
```

**Comment by oxzi** [comment]
@dotlambda: Thanks for your feedback. I altered the PR accordingly.

File: pkgs/development/python-modules/pytmx/default.nix [hunk 1]
```diff
@@ -5,20 +5,27 @@
 
 buildPythonPackage rec {
   pname = "pytmx";
-  version = "3.24.0";
+  version = "3.25.0";
 
   disabled = isPy27;
 
   src = fetchFromGitHub {
     # The release was not git tagged.
```
Comment (dotlambda) [inline pkgs/development/python-modules/pytmx/default.nix:13]: https://github.com/bitcraft/pytmx/issues/147

File: pkgs/development/python-modules/pytmx/default.nix [hunk 2]
```diff
@@ -5,20 +5,27 @@
 
 buildPythonPackage rec {
   pname = "pytmx";
-  version = "3.24.0";
+  version = "3.25.0";
```
Comment (dotlambda) [inline pkgs/development/python-modules/pytmx/default.nix:8]:
```suggestion
  version = "3.25";
```

File: pkgs/development/python-modules/pytmx/default.nix [hunk 3]
```diff
@@ -5,20 +5,27 @@
...
 
   disabled = isPy27;
 
   src = fetchFromGitHub {
     # The release was not git tagged.
     owner = "bitcraft";
     repo = "PyTMX";
-    rev = "eb96efea30d57b731654b2a167d86b8b553b147d";
-    sha256 = "1g1j4w75zw76p5f8m5v0hdigdlva2flf0ngyk8nvqcwzcl5vq5wc";
+    rev = "070680418e832e1d85643e13fcd0f577d7ca6798";
```
Comment (dotlambda) [inline pkgs/development/python-modules/pytmx/default.nix:16]:
```suggestion
    rev = version;
```

**Review by dotlambda — CHANGES_REQUESTED** [review]

**Comment by oxzi** [comment]
@dotlambda: Thanks for bringing the maintainer to tag the releases. I have updated the pull request.

**Pull request merged**